AppCenter/AppCenterTests/MSAppCenterIngestionTests.m

```diff
@@ -574,6 +576,47 @@ - (void)testCompressHTTPBodyWhenNeeded {
   XCTAssertTrue(request.HTTPBody.length < httpBody.length);
 }
 
+- (void)testSendsAuthHeaderWhenAuthTokenIsNotNil {
+
+  // If
+  NSString *expectedToken = @"auth token";
+  MSLogContainer *logContainer = [[MSLogContainer alloc] initWithBatchId:@"whatever" andLogs:(NSArray<id<MSLog>> *)@ [[MSMockLog new]]];
+  self.sut.authToken = expectedToken;
+
+  // When
+  NSURLRequest *request = [self.sut createRequest:logContainer eTag:nil];
+
+  // Then
+  NSString *expectedHeader = [NSString stringWithFormat:kMSBearerTokenHeaderFormat, expectedToken];
+  NSString *actualHeader = request.allHTTPHeaderFields[kMSAuthorizationHeaderKey];
+  XCTAssertEqualObjects(expectedHeader, actualHeader);
+}
+
+- (void)testDoesNotSendAuthHeaderWithNilAuthToken {
+
+  // If
+  MSLogContainer *logContainer = [[MSLogContainer alloc] initWithBatchId:@"whatever" andLogs:(NSArray<id<MSLog>> *)@ [[MSMockLog new]]];
+
+  // When
+  NSURLRequest *request = [self.sut createRequest:logContainer eTag:nil];
+
+  // Then
+  XCTAssertNil([request.allHTTPHeaderFields valueForKey:kMSAuthorizationHeaderKey]);
+}
+
+- (void)testDoesNotSendAuthHeaderWithEmptyAuthToken {
+
+  // If
+  MSLogContainer *logContainer = [[MSLogContainer alloc] initWithBatchId:@"whatever" andLogs:(NSArray<id<MSLog>> *)@ [[MSMockLog new]]];
+  self.sut.authToken  = @"";
```

Comment:
```suggestion
  self.sut.authToken = @"";
```

Reply:
Probably would be fixed by running formatter

Reply:
Yes I forgot to do that on latest commit. Good catch.